packages/cli-core/src/package-manager-controller/package_manager_controller_factory.ts

```diff
@@ -31,7 +32,9 @@ export class PackageManagerControllerFactory {
         if (this.platform === 'win32') {
           const errorMessage =
             'Amplify does not support PNPM on Windows. \nDetails: https://github.com/aws-amplify/amplify-backend/blob/main/packages/create-amplify/README.md';
-          throw new Error(errorMessage);
+          throw new AmplifyUserError('PNPMonWindowsError', {
+            message: errorMessage,
```

Comment:
can use the details field on the constructor rather than putting details in the message

Reply:
Done f70a03a9e6